tests/models/test_image_querying.py

```diff
@@ -538,8 +538,8 @@ def test_image_query(ptf_ref, decam_reference, decam_datastore, decam_default_ca
         stmt = Image.query_images(max_exp_time=60, order_by='quality', seeing_quality_factor=factor)
         results3 = session.scalars(stmt.limit(2)).all()
 
-        # quality will be a higher, but also a different image will now have the second-best quality
-        assert results3 != results1
+        # images stay the same but quality will be higher
+        assert results3 == results1
```

Comment:
The only reason I wanted this second test is to show that we can find a `seeing_quality_factor` that shuffles the order of the images. Maybe if you increase it above `3.0` it would do that?